samza-test/src/test/java/org/apache/samza/test/processor/TestTaskApplication.java

```diff
@@ -81,4 +107,8 @@ public void describe(TaskApplicationDescriptor appDescriptor) {
                  .withOutputStream(outputDescriptor)
                  .withTaskFactory((AsyncStreamTaskFactory) () -> new TestTaskImpl());
   }
+
+  public interface TaskApplicationCallback {
```

Comment:
nit: prefer calling it `TaskApplicationProcessCallback` to be more explicit.

Reply:
Will change.